.all-contributorsrc

```diff
@@ -0,0 +1,37 @@
+{
+  "projectName": "semantic-release-config-logdna",
+  "projectOwner": "logdna",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "angular",
```

Comment:
just to confirm/clarify, this is the convention the cli or bot will use for the commit, not related to any parsing of the contribution type based on commit format.

Reply:
correct